docs/source/examples.rst

```diff
@@ -179,6 +179,9 @@ You need to `setup SageMaker <faq.html#how-can-i-run-on-aws-and-sagemaker>`__
 before being able to use the SageMaker backend. More details are provided
 in `this tutorial <tutorials/basics/basics_backend.html>`__.
 
+This example can be sped up by using SageMaker managed warm pools, as in
+`this example <#sagemaker-backend-and-checkpointing>`__.
```

Comment:
what's the meaning of double underscore?

Reply:
I use double underscore everywhere. I thought this was the correct way to write things?

Reply:
It apparently works as well